omniNotes/src/androidTest/java/it/feio/android/omninotes/tests/HamburgerTest.java

```diff
@@ -0,0 +1,93 @@
+package it.feio.android.omninotes.tests;
+
+import org.junit.Test;
+
+import it.feio.android.omninotes.helpers.EspressoTestBase;
+import it.feio.android.omninotes.screens.Archives;
+import it.feio.android.omninotes.screens.Home;
+import it.feio.android.omninotes.screens.NavigationDrawer;
+import it.feio.android.omninotes.screens.Trash;
+
+import static junit.framework.Assert.assertFalse;
+import static junit.framework.Assert.assertTrue;
+
+public class HamburgerTest extends EspressoTestBase {
+
+    @Test
+    public void testHamburgerValidation() throws InterruptedException {
+        // Step: Click Add button
+        Home.clickAddButton();
+
+        // Step: Click Text Note button
+        Home.clickTextNoteButton();
+
+        // Step: Add note with title "Pay the bills"
+        Home.addPayTheBillsNote();
+
+        // Step: Click return button
+        Home.clickReturnButton();
+
+        // Step: Delete the note with title "Pay the bills"
+        Home.deleteNoteWithTitlePayTheBills();
+
+        // Wait 2 seconds
+        Thread.sleep(2000);
+
+        // Step: Click on hamburger button
+        Home.clickOnHamburgerButton();
+
+        // Wait 2 seconds
+        Thread.sleep(2000);
+
+        // Verify: The button with name Archive is displayed and has sibling the button with name Notes
+        assertTrue("Archive button is not displayed", NavigationDrawer.isArchiveButtonDisplayedAndHasSiblingTheNoteButton());
```

Comment:
The message in case of returning false is a little confusing because the Archive Button can be displayed, but does't have sibling Note Button.

Reply:
Done.